.env.example

```
@@ -8,4 +8,6 @@ VITE_APP_INSIGHTS_KEY=""
 VITE_APP_INSIGHTS_CONNECTION_STRING=""
 VITE_BETTER_AUTH_URL=""
 VITE_AQUIFER_API_URL=""
-VITE_AQUIFER_API_KEY=""
\ No newline at end of file
+VITE_AQUIFER_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)